packages/runtime/container-runtime/src/id-compressor/identifiers.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { OpSpaceCompressedId, SessionSpaceCompressedId } from "@fluidframework/runtime-definitions";
+
+/**
+ * An identifier (v4 UUID) that has been shortened by a distributed compression algorithm.
+ * Lacks a space (session/op), meaning its scope is the same as the space-specific ID from which it was derived.
+ * @alpha
+ */
+export type CompressedId =
+	| SessionSpaceCompressedId
+	| OpSpaceCompressedId
+	| FinalCompressedId
+	| LocalCompressedId;
+
+/**
+ * A compressed ID that is stable and unique within the scope of network of compressors (i.e. a document).
+ * It can only be used/decompressed in the context of the originating document.
+ * @alpha
```

Comment:
Remove all @alpha tags in the whole project

Reply:
done